src/snowflake/cli/_plugins/nativeapp/entities/application_package.py

```diff
@@ -1095,17 +1096,12 @@ def apply_package_scripts(
         )
 
         # once we're sure all the templates expanded correctly, execute all of them
-        with cls.use_package_warehouse(
-            package_warehouse=package_warehouse,
-        ):
-            try:
-                for i, queries in enumerate(queued_queries):
-                    console.step(f"Applying package script: {package_scripts[i]}")
-                    get_sql_executor().execute_queries(queries)
-            except ProgrammingError as err:
-                generic_sql_error_handler(
-                    err, role=package_role, warehouse=package_warehouse
-                )
+        for i, queries in enumerate(queued_queries):
+            script_name = package_scripts[i]
+            console.step(f"Applying package script: {script_name}")
+            get_snowflake_facade().execute_user_script(
```

Comment:
nit: for calls that have no obvious argument order, please use keyword args for extra readability